config/src/defaults.rs

```diff
@@ -133,8 +133,7 @@ pub trait Defaults {
     }
 
     fn consensus_constants_max_block_weight(&self) -> u32 {
-        // TODO: Replace  with real max_block_weight value used in mainnet
-        10_000
+        150_000
```

Comment:
I am a man of powers of two:
```suggestion
        153_600
```